test/Microsoft.AspNet.Mvc.FunctionalTests/RemoteAttributeValidationTest.cs

```diff
@@ -23,12 +23,12 @@ public RemoteAttributeValidationTest(MvcTestFixture<ValidationWebSite.Startup> f
         public HttpClient Client { get; }
 
         [Theory]
-        [InlineData("Aria", "/Aria")]
+        [InlineData("Area1", "/Area1")]
```

Comment:
> The structure of areas was also inconsistent, like we named the value in the attribute as Aria but the folder as `MyArea`

Well, the attributes are far more important than the folders but agree consistency is good. Just thrown by changes that aren't cleanup or consistency improvements in this PR.